packages/pigment-css-react/README.md

```diff
@@ -765,3 +766,85 @@ function App() {
   )
 }
 ```
+
+## RTL Support
+
+Pigment CSS offers built-in mechanism to automatically output corresponding rtl or ltr CSS. If your app by default caters to ltr direction, you also have an option to configure the plugin to output the CSS for the other direction automatically. To configure Pigment CSS for this, update the bundler config.
+
+### Next.js
+
+```js
+const { withPigment } = require('@pigment-css/nextjs-plugin');
+
+// ...
+module.exports = withPigment(nextConfig, {
+  theme: yourCustomTheme,
+  // CSS output option
+  css: {
+    // Specify your default CSS authoring direction
+    defaultDirection: 'ltr',
+    // If you want to output CSS for the direction other than
+    // the `defaultDirection`. Default is `false`.
+    generateForBothDir: true,
+    // Pass this to customize the dir selector. Default is
+    // [dir="rtl"] or [dir="ltr"]
+    getDirSelector(dir: string) {
+      // return your own selector that you'd like to use.
+      return `:dir(${dir})`;
+    }
```

Comment:
Do we need to mention this in the basic demo? I would make the first demo as simple as possible and maybe add after that a section for this option.

Reply:
I had a similar thought. Updated.